bundles/pubsub/pubsub_protocol/pubsub_protocol_wire_v1/gtest/src/PS_WP_tests.cc

```diff
@@ -234,6 +234,7 @@ TEST_F(WireProtocolV1Test, WireProtocolV1Test_DecodeMetadata_Test) { // NOLINT(c
     memcpy(exp + 4, "1:a,1:b,", 8);
 
     pubsub_protocol_message_t message;
+    message.header.convertEndianess = false;
```

Comment:
Endianess => Endianness

Reply:
I will correct this as bug fix, but note that this does change the interfaces (SPI) header.